docs/index.md

```diff
@@ -4,28 +4,21 @@
 
 # Welcome to the EESSI project documentation!
 
-!!! quote
-    What if there was a way to avoid having to install a broad range of scientific software from scratch on every HPC cluster or cloud instance you use or maintain, without compromising on performance?
-
 The **European Environment for Scientific Software Installations** (EESSI, pronounced as "easy") is a collaboration between different European partners in HPC community.
 
 The goal of this project is to build a common stack of scientific software installations for HPC systems and beyond, including laptops,
 personal workstations and cloud infrastructure.
 
-More details about the project are available in the different subsections:
-
-* [Project overview](overview.md)
-* [Filesystem layer](filesystem_layer.md)
-* [Compatibility layer](compatibility_layer.md)
-* [Software layer](software_layer.md)
-* [Pilot repository](pilot.md)
-* [Getting access to EESSI](getting_access/is_eessi_accessible.md)
-* [Using EESSI](using_eessi/setting_up_environment.md)
-* [Software testing](software_testing.md)
-* [Getting support](support.md)
-* [Meetings](meetings.md)
-* [Project partners](partners.md)
-* [Contact info](contact.md)
+Quick links:
+
+* [What is EEESSI?](overview.md)
+* EESSI layered structure: [filesystem](filesystem_layer.md), [compatibility](compatibility_layer.md), [software](software_layer.md)
+* [EESSI pilot software stack and available software](pilot.md)
+* [Access](getting_access/is_eessi_accessible.md), [configure](using_eessi/setting_up_environment.md) and [use](using_eessi/basic_commands.md) EESSI
+* [How to run EESSI test suite](test-suite/index.md)
+* [Contribute software to EESSI](software_layer/adding_software.md)
+* [Get help or report issue](support.md)
+
```

Comment:
We're missing the last three items from the original list here, are these deliberate omissions? I think we at least want to keep the contact link around